src/main/java/TaskList.java

```diff
@@ -0,0 +1,82 @@
+import java.util.ArrayList;
+import java.util.List;
+
+public class TaskList {
+    private static final String indent = "    ";
+    private final List<Task> tasks;
+
+    public TaskList() {
+        this.tasks = new ArrayList<>();
+    }
+
+    public void addTodo(String s) {
+        Todo t = new Todo(s);
+        tasks.add(t);
+        this.printAdd();
+    }
+
+    public void addDdl(String s) {
```

Comment:
I feel that the parameter name is too short, perhaps consider using a more descriptive parameter name? I noticed this issue in the other `add` methods too.
```suggestion
    public void addDdl(String taskDescription) {
```